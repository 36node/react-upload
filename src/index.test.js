import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Upload from "./index";

it("should renders correctly", () => {
  const tree = renderer.create(<Upload>upload</Upload>).toJSON();
  expect(tree).toMatchSnapshot();
});
