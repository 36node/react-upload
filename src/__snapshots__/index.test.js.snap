// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<span
  className=""
>
  <div
    className="ant-upload ant-upload-select ant-upload-select-text"
  >
    <span
      className="ant-upload"
      onClick={[Function]}
      onDragOver={[Function]}
      onDrop={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex="0"
    >
      <input
        accept=""
        directory={null}
        multiple={false}
        onChange={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
        type="file"
        webkitdirectory={null}
      />
      upload
    </span>
  </div>
  <div
    className="ant-upload-list ant-upload-list-text"
  />
</span>
`;
